examples/db_connector/2_postgres_db.py

```diff
@@ -35,22 +41,23 @@
 # )
 
 # ######## postgresql #########
-db_uri = "postgresql://{}:{}@localhost:5432/{}".format(
-    os.getenv("POSTGRES_USERNAME"),
-    os.getenv("POSTGRES_PASSWORD"),
-    os.getenv("POSTGRES_DB"),
-)
-db = connector_factory(db_uri)
-
-
-pipeline = Pipeline.from_script(
-    TOY_SCRIPT,
-    context_storage=db,
-    start_label=("greeting_flow", "start_node"),
-    fallback_label=("greeting_flow", "fallback_node"),
-)
-
-if __name__ == "__main__":
-    check_happy_path(pipeline, HAPPY_PATH)
-    if is_interactive_mode():
-        run_interactive_mode(pipeline)
+#db_uri = "postgresql://{}:{}@localhost:5432/{}".format(
```

Comment:
WHY? Why did we comment it?

Reply:
Because 2_postgres_db and 3_ydb crashed during building. Now this is not commented.